src/controllers/loginController.js

```diff
@@ -167,7 +167,7 @@ class loginController {
       // RETURN USER DETAILS
       const lastUpdatedPassword = findUser.passcodeModifiedAt;
       const thirtyDaysAgo = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000);
-      if (lastUpdatedPassword > thirtyDaysAgo && lastUpdatedPassword !== null) {
+      if (lastUpdatedPassword < thirtyDaysAgo && lastUpdatedPassword !== null) {
```

Comment:
Need to change back to > because this condition checks if the `lastUpdatedPassword` is a date that falls after `thirtyDaysAgo` . it means that the user's password has been updated within the last 30 days and `lastUpdatedPassword `is not null. This condition is used to determine whether to include the **changePassword flag** in the response indicating that the user needs to change their password.